tox.ini

```diff
@@ -1,180 +1,47 @@
 [tox]
-envlist = flake8,
-          py26twisted,
-          py26asyncio,
-          py27twisted,
-          py33twisted,
-          py34twisted,
-          py27asyncio,
-          py33asyncio,
-          py34asyncio,
-          pypy2twisted,
-          pypy2asyncio
+envlist =
+    flake8
+    {pypy,py27,py33}-{twtrunk,trollius}
+    {py34}-{twtrunk,asyncio}
+    {pypy,py26,py27}-{tw121,tw132,twcurrent}
 
 [testenv]
 deps =
-    pytest
-    mock==1.0.1
+    mock
+    unittest2
     coverage
-commands = python -m pytest
+    msgpack-python
+
+    ; twisted dependencies
+    twtrunk: https://github.com/twisted/twisted/archive/trunk.zip
+    tw121: Twisted==12.1
+    tw132: Twisted==13.2
+    twcurrent: Twisted
+
+    ; asyncio dependencies
+    asyncio,trollius: pytest
+    trollius: trollius>=0.1.2
+    trollius: futures>=2.1.5
+commands =
+    sh -c "which python"
+    python -V
+    coverage --version
+    asyncio,trollius: coverage run {envbindir}/py.test autobahn
+    twtrunk,twcurrent,tw121,tw132,twcurrent: coverage run {envbindir}/trial autobahn
+    coverage report
 whitelist_externals = sh
+setenv =
+   asyncio,trollius: USE_ASYNCIO = 1
+   twtrunk,twcurrent,tw121,tw132,twcurrent: USE_TWISTED = 1
 
 
 [testenv:flake8]
+skip_install = True
 deps =
    flake8
 commands =
    sh -c "which python"
    python -V
    flake8 --version
-   flake8 -v --ignore=E501 autobahn
-basepython = python2.7
-
-
-[testenv:py26twisted]
-deps =
-   {[testenv]deps}
-   twisted
-   unittest2
-commands =
-   sh -c "which python"
-   sh -c "which trial"
-   python -V
-   trial --version
-   trial autobahn
-basepython = python2.6
-install_command = pip install {packages} autobahn[twisted,serialization]
```

Comment:
Erm, seems like I'm the dumb one here actually -- this should be the URL: https://github.com/hawkowl/AutobahnPython/blob/more-twisted-tox-versions/tox.ini#L14